internal/handlers/lokistack_create_or_update.go

```diff
@@ -60,13 +60,25 @@ func CreateOrUpdateLokiStack(ctx context.Context, req ctrl.Request, k k8s.Client
 		)
 	}
 
+	var cm corev1.ConfigMap
+	key = client.ObjectKey{Name: manifests.LokiConfigMapName(stack.Name), Namespace: stack.Namespace}
+	if err = k.Get(ctx, key, &cm); err != nil {
+		if !apierrors.IsNotFound(err) {
+			return kverrors.Wrap(err, "failed to lookup lokistack config map", "name", key)
+		}
+	}
+
 	// Here we will translate the lokiv1beta1.LokiStack options into manifest options
 	opts := manifests.Options{
 		Name:          req.Name,
 		Namespace:     req.Namespace,
 		Image:         img,
 		Stack:         stack.Spec,
 		ObjectStorage: *storage,
+		Config: manifests.Config{
+			Config:        cm.BinaryData["config.yaml"],
+			RuntimeConfig: cm.BinaryData["runtime-config.yml"],
+		},
```

Comment:
out of curiosity, why did you opt to add the contents of the CM into the config here vs just pulling it as part of `LokiConfigMap` call? it seems like we don't ever refer back to this data except for when we do our diff comparison...